usvm-ts/src/test/kotlin/org/usvm/samples/FieldAccess.kt

```diff
@@ -77,8 +77,8 @@ class FieldAccess : TsMethodTestRunner() {
                 x.number == 1.1 && r.number == 14.0
             },
             { a, r ->
-                val x = a.properties["x"] as TsTestValue.TsNumber
-                x.number != 1.1 && r.number == 10.0
+                val x = a.properties["x"] as? TsTestValue.TsNumber
```

Comment:
Why "as?" cast? We expect `a.properties["x"]` to be `number` in all execution branches, so we cast it in both properties that we check. Why changing it to `as?` cast only in one?

Reply:
Why? The source code: 
```
    conditionalFieldAccess(a: SimpleClass): number {
        if (a.x === 1.1) return 14;
        return 10;
    }

class SimpleClass {
    x: any = 5;
}

```

x has `any` type, for example, undefined can be chosen as a parameter for a test